src/command/identity/export.ts

```diff
@@ -0,0 +1,77 @@
+import { Argument, LeafCommand } from 'furious-commander'
+import { exit } from 'process'
+import { Identity, IdentityType } from '../../service/identity/types'
+import { RootCommand } from '../root-command'
+import { printNoIdentitiesError, promptUserForIdentity } from './common'
+
+export class Export extends RootCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'export'
+
+  public readonly description = 'Export identity'
+
+  @Argument({ key: 'identity-name', describe: 'Name of the identity to be exported' })
+  public identityName!: string
+
+  public async run(): Promise<void> {
+    this.initCommand()
+    this.checkForNoIdentities()
+    const options = this.getV3Identities()
+    this.checkForEmptyIdentities(options)
+    await this.ensureIdentityNameIsProvided(options)
+    this.checkForExistingIdentity()
+    const identity: Identity = this.commandConfig.config.identities[this.identityName]
```

Comment:
If you use the `getWalletFromIdentity` function here it returns a `Wallet` which has a `toV3String` function to create a V3 JSON. Then it would be possible to export the `privateKey` based identities as well.

Maybe it's worth considering or for a next PR.

Reply:
I don't think it's a good idea, because those are disposable entities without password only should work in a Swarm session ideally.
Also as I know for V3 format a password is neccessary in order to encrypt private key, thereby you have to pass it and the output won't be the same as the source.

Reply:
I see some scenarios where a disposable keypair ends up being used more than it was originally supposed to, and then it would be beneficial to convert it to a secure V3 wallet.

We can use something like `Wallet.fromPrivateKey(privateKey).toV3String(password)` with a new password the user has to confirm.

Not sure if this is the scope of the first iteration, but I support developing this in the near future.

Reply:
> We can use something like `Wallet.fromPrivateKey(privateKey).toV3String(password)` with a new password the user has to confirm.

I think that's a good direction.

> Not sure if this is the scope of the first iteration, but I support developing this in the near future.

I agree, let's keep it simple for now and add this as a feature later.